README.md

```diff
@@ -31,6 +31,7 @@ android {
 androidEmulator {
     emulator {
         name 'my_avd_emulator_name' // Defaults to be dynamically based on the configuration of the AVD
+        device 'pixel_xl' // or device '19'. For options, run avdmanager list device
```

Comment:
Could you follow the convention of the rest of the arguments and explain what the default behavior is in addition to what you have now?

Reply:
How about "Defaults to exclude the device flag, using avdmanager default. For options, run avdmanager list device"?

Reply:
Yup, that works for me